src/driver/linux_resource/kernel_compat_funcs.sh

```diff
@@ -329,7 +329,7 @@ function test_export()
     #        May give a false positive if the export is conditional.
     #     3. The MAP file if present. May give a false positive
     #        because it lists all extern (not only exported) symbols.
-    if [ -f $KBUILD_MODULE_SYMVERS ]; then
+    if [[ $KBUILD_MODULE_SYMVERS && -f $KBUILD_MODULE_SYMVERS ]]; then
```

Comment:
Sorry I was concentrating on whether bashisms were acceptable when I accepted this review but I think it's a bit messy duplicating the variable _and_ relying on not explicit behaviour, as this isn't specifically checking for being set.
I would prefer just putting the expansion in `"` in this case.